config/config.go

```diff
@@ -109,7 +109,7 @@ type DaemonConfig struct {
 }
 
 type LoggingConfig struct {
-	LogToStdout         bool
+	LogToStdout         bool   `toml:"log_to_stdout"`
```

Comment:
@sctb512 The `log_to_stdout` config is already restored

Reply:
Yes, it does. But the `log_to_stdout` entry in `misc/snapshotter/config.toml` is lost.

Reply:
Yes. another PR should be raised to fix the example config toml